hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/operations/ReadManyOperation.java

```diff
@@ -155,6 +135,10 @@ protected void readInternal(ObjectDataInput in) throws IOException {
         minSize = in.readInt();
         maxSize = in.readInt();
         filter = in.readObject();
-        returnPortable = in.readBoolean();
+        // RU_COMPAT_3_9
+        if (in.getVersion().isLessThan(V3_10)) {
```

Comment:
Should we use `isUnknownOrLessThan` here? We can use the same one in `writeInternal` for consistency.

Reply:
good catch!